share/models/logs.py

```diff
@@ -214,6 +215,9 @@ class HarvestLog(AbstractBaseLog):
 
     class Meta:
         unique_together = ('source_config', 'start_date', 'end_date', 'harvester_version', 'source_config_version', )
+        indexes = [
+            ConcurrentIndex(fields=['harvester_version'])
```

Comment:
Should there be a migration for this, or are you saving that for another (janitor) PR?